torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -221,6 +221,52 @@ const std::vector<std::string> functions = {
 
             # FIXME: torchscript: torch.zeros(sizes, grad.options())
             return torch.zeros(sizes).to(grad).scatter_(dim, indices, grad)
+        
+        # python implementation of at::_gather_sparse_backward function. Need to Refine
+        def AD_gather_sparse_backend(grad,
```

Comment:
this looks identical to what's in `_gather_sparse_backward`. Is there any good reason that we are re-doing this in python?

Reply:
Just because of `gather_backward` is an `aten` function, not `torch`.